examples/amp-story/branching.html

```diff
@@ -0,0 +1,78 @@
+<!doctype html>
```

Comment:
`amp-story-branching.html`?

Reply:
We don't really follow this practice in this folder. Should we start using it?

Reply:
Ah right, that's fine :)